webrender/src/device/gfx.rs

```diff
@@ -964,14 +881,84 @@ impl<B: hal::Backend> UniformBuffer<B> {
     }
 }
 
+pub struct VertexBufferHandler<B: hal::Backend> {
+    buffer: Buffer<B>,
+    memory_types: Vec<hal::MemoryType>,
+    usage: hal::buffer::Usage,
+    data_stride: usize,
+    pitch_alignment: usize,
+    pub buffer_len: usize,
```

Comment:
why does it need to be `pub`?

Reply:
Only for the read_pixels. The requested buffer length can change. (e.g. alignment)
https://github.com/szeged/webrender/pull/206/files/3a828c154732276c863362d26ee430078e64fa4f#diff-4d269bbd98df6af055b56ab19362567fR3253